integration/spark/src/main/common/java/io/openlineage/spark/agent/lifecycle/plan/CreateDataSourceTableAsSelectCommandVisitor.java

```diff
@@ -1,11 +1,16 @@
 package io.openlineage.spark.agent.lifecycle.plan;
 
```

Comment:
Please take a look at the integration tests in `SparkContainerIntegrationTest.java`. 
Perhaps some of them can be extended to verify if OpenLineage event contains `tableStateChange` facet.